spec/index.html

```diff
@@ -943,6 +943,15 @@ <h3>Converting a <a>YAML scalar</a></h3>
               the conversion result is a <a>language-tagged string</a>
               with value taken from |n|,
               ane a <a>language tag</a> taken from the suffix of |t|.
+              <div class="note">
+                <a>Node tags</a> including an underscore (`"_"`),
```

Comment:
I went back and forth on this. The JSON-LD specs use `"_"` instead of "`_`" for equivalent quoted characters (e.g., in https://www.w3.org/TR/json-ld11/#the-i18n-namespace) That may make it more readable, if not be as accurate. If we do make the change, it needs to be done comprehensively, so I'll leave this out of this batch.

Reply:
Fair enough. There may be some W3-ish style guide or examples that provides guidance, or others in the WG with opinions. I can live with the `"_"` style if there's general agreement that it's better, though it's not to my taste.

Reply:
There were about four points in the document that use this style for "_", "#", and "%23". To my mind it's easier to see these as it exists easily, given how the spec styles code blocks.